### ISSUE TITLE
Tweak List empty, global black and whitelist not working (iOS 10.2)

### ISSUE THREAD

**Issue by emcepe** [opa334/Choicy#1]
Hi Lars,

I REALLY like your tweak :)

I've been waiting a long time for a tweak like this, thank you!

I'd like to see a feature where you can select a global blacklist and/or whitelist. I have lots of tweaks and many where I have to blacklist same some tweaks - so with a global black-/whitelist its easier and if theres an App which needs changes one can change the settings per App.

Thank you!

**Comment by opa334** [comment]
I will look into this in the future!

**Comment by opa334** [comment]
Added in 1.1.

**Comment by emcepe** [comment]
Cool. Thank you very much.
But unfortunately it does not work for me :-(
I've tried it with global disabling EmojiPort and whitelisting it for Threema for example but it won't load at all.

**Comment by emcepe** [comment]
And tweaks like CrashReporter or iCleanerPro or Wifi Explorer and some more have an empty tweak list with v1.1

**Comment by opa334** [comment]
You're right, something messed up. Looking into it now.

**Comment by opa334** [comment]
No, I was wrong. It does work. The issue I was having was related to something different. Make sure "Allow whitelist overrides" is enabled.

**Comment by emcepe** [comment]
I am sure that the option is set. I double-checked all before writing here.

The other problem is that CrashReporter, iCleanerPro, Wifi Explorer and some more have an empty tweak list, and I think it has nothing to do with that option :)

**Comment by opa334** [comment]
I reproduced empty tweak list issue, it's a small oversight on 32 bit devices. Will be fixed shortly. The first issue I cannot reproduce however, on my devices it works.

**Comment by emcepe** [comment]
Hmm a/my 6s isn't 32-Bit device.

Well, maybe it has something to do with the other issue I reported.

**Comment by emcepe** [comment]
I've tried 1.1.1-2 but the problems remain:
- Tweak list is empty for given apps/tweaks.
- global blacklist and custom whitelist does not work for me on 10.2

**Comment by opa334** [comment]
Both work fine for me on 10.3.3

**Comment by opa334** [comment]
I will post a debug build to get some logs tomorrow... Maybe that could help me nail down the issue.
